pkg/action/action.go

```diff
@@ -264,6 +266,20 @@ func (a *Action) ValidateInput(inp Input) error {
 	return nil
 }
 
+// ValidateInput validates input arguments in action definition.
+func (a *Action) ValidateInput(isExec bool) error {
```

Comment:
No, you shouldn't check isExec here.
Action shouldn't know about container environment flags.